router/image/templates/nginx.conf

```diff
@@ -15,43 +15,47 @@ http {
     sendfile on;
     tcp_nopush on;
     tcp_nodelay on;
-
     # The Timeout value must be greater than the front facing load balancers timeout value.
     # Default is the deis recommended timeout value for ELB - 1200 seconds + 100s extra.
-    {{ $defaultTimeout := or (.deis_router_defaultTimeout) "1300" }}
```

Comment:
Something like `get` which returns the key or an empty string if not found. That way we can go `{{ $defaultTimeout := or (get "/deis/router/defaultTimeout") "1300" }}`